src/main/java/racingcar/Cars.java

```diff
@@ -31,4 +28,22 @@ private void createCars(int carCount) {
     public int getCarsSize() {
         return cars.size();
     }
+
+    public Cars carMove() {
+        for(Car car : cars) {
+            car.move(new RandomNumberMovingStrategy(RandomGenerator.getInstance()));
+        }
+        return this;
+    }
+
+    public List<Car> findWinner() {
+        Car winner = cars.get(0);
+        List<Car> winners = new ArrayList<>();
+        for(Car car : cars) {
+            if(car.getPosition() > winner.getPosition() || car.getPosition() == winner.getPosition()) {
```

Comment:
```suggestion
            if (car.getPosition() > winner.getPosition() || car.getPosition() == winner.getPosition()) {
```
그리고 `if` 조건도 `method`로 추출되면 가독성 측면에서 더 좋을 것 같습니다 😄 
그리고 람다식을 활용하면 이 부분도 개선이 될 수 있는데요 
```java
cars.stream()
        .mapToInt(Car::getPosition)
        .max();
```
이와 같이 1등을 구해볼 수 있을 것 같습니다 😄 
이 부분은 참고만 부탁 드리겠습니다 🙇

Reply:
> 그리고 `if` 조건도 `method`로 추출되면 가독성 측면에서 더 좋을 것 같습니다 😄
> 그리고 람다식을 활용하면 이 부분도 개선이 될 수 있는데요
> 
> ```java
> cars.stream()
>         .mapToInt(Car::getPosition)
>         .max();
> ```
> 
> 이와 같이 1등을 구해볼 수 있을 것 같습니다 😄
람다식 사용하기가 처음엔 많이 어색했는데 직접 사용을 해봐니까 조금 나아진거같습니다 ㅎㅎ 
람다식으로 적용해 보겠습니다. 감사합니다!

Reply:
람다식 사용하기가 처음엔 많이 어색했는데 직접 사용을 해봐니까 조금 나아진거같습니다 ㅎㅎ
람다식으로 적용해 보겠습니다. 감사합니다!